f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py

```diff
@@ -651,8 +674,11 @@ def update_loadbalancer(self, context, old_loadbalancer,
     def delete_loadbalancer(self, context, loadbalancer, service):
         """Handle RPC cast from plugin to delete_loadbalancer."""
         try:
-            self.lbdriver.delete_loadbalancer(loadbalancer, service)
+            service_pending = \
+                self.lbdriver.delete_loadbalancer(loadbalancer, service)
```

Comment:
You are right, all deletes will return 'False'  So we probably don't need to check.  I did it for consistency.